packages/metro-config/src/defaults/index.js

```diff
@@ -65,9 +65,11 @@ const getDefaultValues = (projectRoot: ?string): IntermediateConfigT => ({
   },
   transformer: {
     assetPlugins: [],
-    asyncRequireModulePath: 'metro/src/lib/bundle-modules/asyncRequire',
+    asyncRequireModulePath: require.resolve(
+      'metro/src/lib/bundle-modules/asyncRequire',
+    ),
     assetRegistryPath: 'missing-asset-registry-path',
-    babelTransformerPath: 'metro/src/defaultTransformer',
+    babelTransformerPath: require.resolve('metro/src/defaultTransformer'),
```

Comment:
We want to avoid using absolute paths for config params in Metro. One of the reasons is that they cause invalidations to our remote caching system (also `asyncRequireModulePath` is used to add a `require()` statement to that specific module so we want to keep it relative).